package.json

```diff
@@ -55,67 +55,68 @@
-    "@commitlint/cli": "^16.0.1",
-    "@commitlint/config-conventional": "^16.0.0",
-    "@nrwl/cli": "13.8.5",
-    "@nrwl/nx-cloud": "13.1.6",
-    "@nrwl/tao": "13.8.5",
-    "@nrwl/workspace": "13.8.5",
-    "@swc/core": "^1.2.159",
-    "@swc/jest": "^0.2.20",
+    "@babel/code-frame": "^7.16.7",
+    "@babel/eslint-parser": "^7.17.0",
+    "@babel/parser": "^7.17.0",
+    "@babel/types": "^7.17.10",
+    "@commitlint/cli": "^16.2.4",
+    "@commitlint/config-conventional": "^16.2.4",
+    "@nrwl/cli": "14.1.4",
+    "@nrwl/nx-cloud": "14.0.3",
+    "@nrwl/tao": "14.1.4",
+    "@nrwl/workspace": "14.1.4",
+    "@swc/core": "^1.2.181",
+    "@swc/jest": "^0.2.21",
     "@types/babel__code-frame": "^7.0.3",
     "@types/debug": "^4.1.7",
     "@types/eslint-visitor-keys": "^1.0.0",
     "@types/glob": "^7.2.0",
     "@types/is-glob": "^4.0.2",
-    "@types/jest": "^27.0.2",
+    "@types/jest": "^27.5.0",
     "@types/jest-specific-snapshot": "^0.5.5",
-    "@types/lodash": "^4.14.176",
-    "@types/marked": "^3.0.2",
+    "@types/lodash": "^4.14.182",
+    "@types/marked": "^4.0.3",
     "@types/ncp": "^2.0.5",
-    "@types/node": "^16.11.4",
-    "@types/prettier": "^2.4.2",
+    "@types/node": "^17.0.31",
+    "@types/prettier": "^2.6.0",
     "@types/rimraf": "^3.0.2",
     "@types/semver": "^7.3.9",
-    "@types/tmp": "^0.2.2",
+    "@types/tmp": "^0.2.3",
     "all-contributors-cli": "^6.20.0",
     "cross-env": "^7.0.3",
-    "cross-fetch": "^3.1.4",
-    "cspell": "^5.12.3",
+    "cross-fetch": "^3.1.5",
+    "cspell": "^5.20.0",
     "cz-conventional-changelog": "^3.3.0",
     "downlevel-dts": "^0.9.0",
-    "enhanced-resolve": "^5.8.3",
-    "eslint": "^8.7.0",
+    "enhanced-resolve": "^5.9.3",
+    "eslint": "^8.15.0",
     "eslint-plugin-eslint-comments": "^3.2.0",
-    "eslint-plugin-eslint-plugin": "^4.0.1",
-    "eslint-plugin-import": "^2.25.2",
-    "eslint-plugin-jest": "^26.0.0",
+    "eslint-plugin-eslint-plugin": "^4.1.0",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jest": "^26.1.5",
     "eslint-plugin-simple-import-sort": "^7.0.0",
     "glob": "^8.0.1",
     "husky": "^8.0.1",
-    "jest": "^27.3.1",
-    "jest-diff": "^27.3.1",
-    "jest-snapshot": "^27.3.1",
+    "jest": "^28.1.0",
+    "jest-diff": "^28.1.0",
+    "jest-snapshot": "^28.1.0",
     "jest-specific-snapshot": "^5.0.0",
     "lerna": "^4.0.0",
-    "lint-staged": "^12.0.2",
+    "lint-staged": "^12.4.1",
     "make-dir": "^3.1.0",
-    "markdownlint-cli": "^0.29.0",
+    "markdownlint-cli": "^0.31.1",
     "ncp": "^2.0.0",
     "prettier": "2.5.1",
-    "pretty-format": "^27.3.1",
+    "pretty-format": "^28.1.0",
     "rimraf": "^3.0.2",
     "tmp": "^0.2.1",
-    "ts-node": "^10.4.0",
+    "ts-node": "^10.7.0",
     "tslint": "^6.1.3",
     "typescript": ">=3.3.1 <4.7.0"
   },
   "resolutions": {
-    "@types/node": "^16.11.4",
-    "typescript": "4.6.2"
+    "@types/node": "^17.0.31",
+    "pretty-format": "^28.1.0",
```

Comment:
one of the jest packages depends on the old pretty-format version which was breaking the types.